ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-release-4.8.yaml

```diff
@@ -44,8 +44,7 @@ tag_specification:
   namespace: ocp
 tests:
 - as: verify
-  commands: |
-    make vendor && git diff --stat HEAD --ignore-submodules --exit-code
+  commands: make verify
```

Comment:
The one downside of making this change is that updating the root Makefile in the openshift/operator-framework-olm repository can be cumbersome due to the release label restrictions when in feature/code freeze, and having to run the other checks as well. If we defined all of the checks here, we aren't gated on needing things like the `valid-bug` label for updating our CI checks.